Runtime/Conditions/TimeoutCondition.cs

```diff
@@ -21,7 +21,7 @@ public class EntityData : IConditionData
             /// The delay before the condition completes.
             /// </summary>
             [DataMember]
-            [DisplayName("Wait for seconds")]
+            [DisplayName("Wait (in seconds)")]
```

Comment:
Perhaps something clearer like "Wait (time in seconds)" 🤔

Reply:
I would like to keep it consistent <shortest name possible> (in <unit>)

Reply:
Alright, but this does not make much sense 😕